public/app/plugins/panel/barchart/TickSpacingEditor.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { SelectableValue, StandardEditorProps } from '@grafana/data';
+import { Checkbox, HorizontalGroup, RadioButtonGroup, Tooltip } from '@grafana/ui';
+
+const GAPS_OPTIONS: Array<SelectableValue<number>> = [
+  {
+    label: 'None',
+    value: 0,
+    description: 'Show all tick marks',
+  },
+  {
+    label: 'Small',
+    value: 100,
+    description: 'Require 100px spacing',
+  },
+  {
+    label: 'Medium',
+    value: 200,
+    description: 'Require 200px spacing',
+  },
+  {
+    label: 'Large',
+    value: 300,
+    description: 'Require 300px spacing',
+  },
+];
+
+export const TickSpacingEditor: React.FC<StandardEditorProps<number, any>> = (props) => {
+  let value = props.value ?? 0;
+  const isRTL = value < 0;
+  if (isRTL) {
+    value *= -1;
+  }
```

Comment:
Nothing that blocks this PR from being merged, but I'm not a huge fan of relying on the value when configuring RTL.

Reply:
> but I'm not a huge fan of relying on the value when configuring RTL

Any alternative ideas?  everything I can think of adds up to the same thing